script/Ciel/Day 13/13_01_01_CIEL13_4.txt

```diff
@@ -504,98 +538,106 @@ That alone is the undeniable truth.
 [sha:81c548f1fdfcf667bef2c91e15c1ce1c58bb5afc]{
 -- Page 338, Offset 34194.
 -- 先輩と出会って二週間にも満たない時間だったけど、本当に―――あんなに生きていて嬉しかった事は、なかったんだ。
-It hasn't even been two weeks since we first met, but even then―――this is the happiest I've been in my entire life.
+It hasn't even been two weeks since we first met, and yet――this is the happiest I've been in my entire life.
 }
 [sha:e0d22961b8f0925898857d2687e90a6e97705f17]{
 -- Page 339, Offset 34195.
 -- 「……ちく、しょう――――」
-"...God, damnit――――"
+"God... damnit―――"
 }
 [sha:75fd6c2cf65ade8b98facfe6c36d3efaaf401d16]{
 -- Page 339, Offset 34196.
 -- でも、それは俺だけの幻想だった。
-But... those were all just my one-sided misapprehensions.
+Except... I was living in a fantasy.
 }
 [sha:075ddf0ceb2ebc30e85f7ad5c9446d96d56823eb]{
 -- Page 339, Offset 34197.
 -- ……目を閉じてしまいそうだ。
-...I feel like my eyes are about to close.
+My eyelids are beginning to droop...
 }
 [sha:4b127649f44905758c81ba73600650924323c036]{
 -- Page 339, Offset 34198.
 -- 憎しみでも怒りでもなく、ただ悔しくて、こんなにも胸が痛んでいる。
-I don't feel any hatred or anger. All I feel is regret and the excruciating pain in my chest.
+I feel no hatred, no anger;
+ only a regret so intense it ties my stomach into knots.
 }
 [sha:4828faac957540f24fc1dcefbe8e6364a14ac37e]{
 -- Page 340, Offset 34199.
 -- 視界が滲んでいく。
-My vision is getting blurry.
+My vision is starting to blur.
 }
 [sha:85882482b1a30e16bb4cea7e964b874ee0614d7e]{
 -- Page 340, Offset 34200.
 -- 頭上には煌々と輝く白い月。
-The dazzling white moon shines high in the sky.
+Far above me, the moon gleams a radiant white.
 }
 [sha:156b017468b7e11872dc6bbde9df130027d77972]{
 -- Page 340, Offset 34201.
 -- 灰色の瓦礫で埋まった静かの海。
-A tranquil sea, surrounded by grey rubble.
+// Assuming this is filled -> comprised of for 埋まる
+Below, a tranquil ocean of grey rubble.
 }
 [sha:e4d3106bc7695e30a54f06e95c41dc6e719eca99]{
 -- Page 340, Offset 34202.
 -- ゆらゆらと揺らいでいる。
-The moon is swaying and wavering.
+Everything is swaying and flickering.
 }
 [sha:4c11e69126bba185ae381bea6fb514f2cc1a84fb]{
 -- Page 340, Offset 34203.
 -- 何もかも偽りで、近づけば消えてしまう幻。
-With everything veiled in lies, it's an illusion that vanishes when you try to approach it.
+It's all just lies; an illusion that vanishes when you try and get too close.
 }
 [sha:99593aeee6019fa1d1feff11f3c09ffd0feaa731]{
 -- Page 340, Offset 34204.
 -- それは一つの、掴みようのない蜃気楼のようだった。
-It's just like a lone mirage that remains out of my reach.
+A mirage that will forever remain out of reach.
```

Comment:
Da titledrop.
Maybe put it last and give the sentence some more oomph? It should pop.

Something forever out of reach, as if a no more than a lone mirage.

Reply:
Hmm, it feels a bit labored / kinda hides the noun behind the qualifiers imo, would probably prefer current or something perhaps radically simpler, e.g.
> A singular, elusive, mirage.

Reply:
Yeah, probably the right call to shrink it down. 掴みようのない can easily be rendered in a whole host of adjectivals. I think elusive perhaps is a bit too vague, though...
>Almost as if an unobtainable mirage.

Other suggestions... unattainable, out-of-reach, uncatchable, ungraspable... there are tons of alternatives. I think perhaps out-of-reach is one of the more interesting alternatives.
As for the things that can be pruned, I'd rather be rid of それは一つの than のようだった, since that binds into the earlier stuff.

Reply:
looking at it again, the current version still feels best. "Almost like an out-of-reach mirage" is just waaay less lyrical than 
> A mirage that will forever remain out of reach.

and given EN sentence structuring, having it up front makes the word read with more oomph to me anyway

Reply:
I very much disagre that it "feels best". I still think my initial suggestion is the best.
If you want to use that structure, I'd still say you need to linking word. that's important.

>Almost as if a mirage, forever out of reach.